src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -223,14 +223,8 @@ public function startTest($test)
                 $test->getTestResultObject()->addError($test, new $AssertionFailedError('`@expectedDeprecation` annotations are not allowed at the class level.'), 0);
             }
             if (isset($annotations['method']['expectedDeprecation'])) {
-                if (!in_array('legacy', $groups, true)) {
-                    $this->error = new $AssertionFailedError('Only tests with the `@group legacy` annotation can have `@expectedDeprecation`.');
-                }
-
-                $test->getTestResultObject()->beStrictAboutTestsThatDoNotTestAnything(false);
```

Comment:
I've moved this to end tests but because we reset the value using $this->reportUselessTests this code does not seem to have the intended affect and we still to add <code>$this->addToAssertionCount(1);</code> to the test to make it not risky.